downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,498 @@
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.5.0](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.5.0%22+label%3AType%2FBug).
+
+## Code to Cloud updates
+
+### Improvements
+- Improved the Kubernetes Horizontal autoscaler to use `autoscaling/v2`.
+
+## Developer tools updates
+
+### New features
+
+#### Language Server
+
+- Added quick fix support for the `Extract to local variable` and `Extract to function` code actions.
+- Added completion support for regular expressions.
+
+#### Persist Tool
+
+>**Info:** The Ballerina persistence feature is an experimental feature. The commands associated with the tool might change in future releases.
+
+- Added the following new arguments to the `persist init` commands.
+    - `--datastore` - This is used to indicate the preferred database client. Currently, only `mysql` is supported.
+    - `--module` - This is used to indicate the module in which the files are generated.
+
+    For example, 
+    ```bash
+    $ bal persist init --datastore mysql --module db
+    ```
+- Changed the `persist init` command to create a `persist` directory in the Ballerina project and generate a new definition file (`model.bal`) in the `persist` directory if the file does not exist.
+- Restricted to have only one persist model definition per Ballerina package.
+- Removed the `persist push` command support and generate SQL script file in the `persist generate` command. The generated SQL script file needs to be executed manually to create the database table.
+- Changed the `persist generate` command to generate all the Ballerina client, types, `db_config` files, and the SQL script files in the `generated/<module_name>` directory.
+- Renamed the generated files to the following.
+  - `generated_client.bal` -> `persist_client.bal`
+  - `generated_types.bal` -> `persist_types.bal`
+  - `database_configuration.bal` -> `persist_db_config.bal`
+  - `<schema_name>_db_script.sql` -> `script.sql`
+- Changed the code in the generated Ballerina client and types files to support the new changes.
+
+### Improvements
+
+#### Bindgen Tool 
+
+- Added the `--with-optional-types`, `--with-optional-types-param`, and `--with-optional-types-return` command options to the `bal bindgen` command, to support handling Java null values via generated bindings. These command options will generate optional (i.e., nilable) types 
+  for parameter or/and return types in generated Ballerina binding functions.
```

Comment:
Unnecessary line break? Check other places too.

Reply:
These will be fixed when the website page is rendered.

Reply:
Same comment about maintaining the MD file.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6740.